presto-benchmark/src/main/java/com/facebook/presto/benchmark/HandTpchQuery6.java

```diff
@@ -92,6 +92,18 @@ public int process(ConnectorSession session, Page page, int start, int end, Page
             return position;
         }
 
+        @Override
+        public Page processColumnar(ConnectorSession session, Page page, List<? extends Type> types)
```

Comment:
Document that these method should return `null` when page is completely filtered out.  Add `@Nullable` to the method declaration.